### ISSUE TITLE
Suggest a different practice around passwords

### ISSUE THREAD

**Issue by bhirsch70** [ansible/lightbulb#103]
Currently, aws_lab_setup both stores the participants passwords in inventory, ```<username>-instances.txt``` as well as sends it to the participant via sendgrid email.

I would suggest removing that password from both the inventory and not sending it out via email.

A workshop instructor can simply tell participants the password on the day of the workshop/training.

**Comment by tima** [comment]
Yeah I agree. That template/method is something leftover from a previous generation of our lab provisioner that needs to be cleaned up. Since we create and copy each students' static inventory into their home directory, emailing it is entirely unnecessary. We just need to email their username and IP address to their control box. 

Removing the static inventory password in the file we upload is something I'd like to come back to. That needs to be in there for the second workshop before the deck gets into what that inventory file is.
